### PR TITLE
Feature/start fresh

### PR DESCRIPTION
**Pull request by teenie-quaggard**
It's a new week and a new tic-tac-toe!

### CODE REVIEW

File: lib/play.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        # tick(@board, @player)
+    end
+
+    def make_move(board, mark, user_input)
+        board.make_move(mark, user_input)
+    end
+
+    def winner(board, mark)
+        tiles = board.tiles
+        indices = tiles.each_index.select{|i| tiles[i] == mark}
```
Comment (martingaston) [inline lib/play.rb]: Does this work as intended if a `mark` is in indices `0,1,2` and, say, `7`?

```
X | X | X
4 | 5 | 6
X | 8 | 9
```

File: lib/play.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    def game_play()
+        @output.prompt_turn
+        @output.print_board(board)
+        user_input = gets.chomp()
+        make_move(board, player.mark, user_input)
+    end
+
+
+    # def tick(board, player)
```
Comment (martingaston) [inline lib/play.rb]: Purely my opinion, but my spidey-sense tingles when I see a PR with commented out dead code. Think it's fine in this instance, though 👍

**Review by martingaston — COMMENTED** [review]
Looks like a lot of positive progress in the achieving the requirements!

File: lib/play.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
+class Play
+    WINNING_INDICES = [
```
Comment (ellehallal) [inline lib/play.rb:2]: What do you think about `Board` containing its winning indices?

File: spec/board_spec.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+RSpec.describe Board do
+    it 'can be initialized with an array of spaces' do
+        board = Board.new()
+        board2 = Board.new(:tiles => [1,2,3,4,5,6,7,8,9,10,11,12,13,14])
+        expect(board.tiles).to eq([1,2,3,4,5,6,7,8,9])
+        expect(board2.tiles).to eq([1,2,3,4,5,6,7,8,9,10,11,12,13,14])
+    end
+
+    it '#make_move replaces number in the tile array with a marker' do
```
Comment (ellehallal) [inline spec/board_spec.rb:11]: 👍🏽nice

File: lib/board.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        @tiles = args[:tiles] || BOARD_RESET
+    end
+
+    def make_move(mark, user_input)
+        digit = user_input.to_i
+        index = digit - 1
+        @tiles[index] = mark
+    end
+
+    def new_board(reset_array)
```
Comment (ellehallal) [inline lib/board.rb:15]: In what scenario would the board tiles need to be reset?

File: spec/output_spec.rb [hunk 6]
```diff
@@ -1,28 +1,51 @@
...
+            console = StringIO.new
+            output = Output.new(:console => console)
+            output.hello
+            expect(console.string).to eq("Howdy stranger. Welcome to Noughts and Crosses.\n")
+        end
     end
-  end
 
+    describe "#get_input" do
+        it "accepts input from a user" do
```
Comment (ellehallal) [inline spec/output_spec.rb:15]: 👍🏽

File: spec/output_spec.rb [hunk 7]
```diff
@@ -1,28 +1,51 @@
...
+    describe "#prompt_turn" do
+        it "prints intro using test double" do
+            console = StringIO.new
+            output = Output.new(:console => console)
+            output.prompt_turn
+            expect(console.string).to eq("\nChoose a number between 1-9:\n")
+        end
+    end
 
+    describe "#winner" do
```
Comment (ellehallal) [inline spec/output_spec.rb]: Is there a difference between this test and the one above?

**Review by ellehallal — COMMENTED** [review]
Looks good :) I've left a few comments

**Pull request merged**